src/nxt_http_parse.h

```diff
@@ -96,11 +96,8 @@ struct nxt_http_field_s {
 
 
 typedef struct {
-    u_char                    *pos;
     nxt_mp_t                  *mem_pool;
-
     uint64_t                  chunk_size;
-
```

Comment:
I would normally suggest such a change (removing the blank lines) be done in a separate commit, but in this case I think it's OK...